pootle/templates/browser/_actions.html

```diff
@@ -26,10 +29,15 @@
     </span>
     {% endif %}
   </li>
+  {% endif %}
   <li id="js-action-continue">
     {% if url_action_continue %}
     <a class="continue-translation" href="{{ url_action_continue }}">
+      {% if can_suggest_only %}
+      <span class="caption">{% trans 'Continue suggesting' %}</span>
```

Comment:
I think that it makes sense to use a continue URL that points to "first units with no suggestions" for users that can only suggest.